link_name: base_footprint

linear_scale : 1.0
angular_scale: 1.0

holonomic: false
